src/mount.ts

```diff
@@ -261,6 +264,33 @@ export function mount(
     to.appendChild(el)
   }
 
+  function slotToFunction(slot: Slot): Function {
```

Comment:
nit: Function is rarely used as a type. We usually prefer `() => string` for example.

Reply:
Ok, I will change this. I suppose it'll be like `type SlotLike = () => string | (() => string) | { render: () => string }` or something to that meaning.

Reply:
The type here is *very* complicated, it wants me to specify some very large union of public component instance, functional components... I just removed the entire return type and will let TS figure it out.